lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -229,6 +266,15 @@ static int at_parse_process_element(const char **str, int index,
 			tmpstr++;
 		}
 
+		at_params_string_put(list, index, start_ptr,
+				     tmpstr - start_ptr);
+	} else if (state == CLAC) {
+		const char *start_ptr = tmpstr;
+
+		while (!is_terminated((int)*tmpstr)) {
```

Comment:
I don't think you need to cast to `int`, this function accepts `char`.

Reply:
Fixed

Reply:
The `(int)` cast is still present.

Reply:
Sorry, missed that one. Fixed.